scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -975,98 +987,113 @@ task SumFloats {
-        else
-            # Extract the tar.gz:
-            mkdir datasources_dir
-            tar zxvf ${data_sources_tar_gz} -C datasources_dir --strip-components 1
-            DATA_SOURCES_FOLDER="$PWD/datasources_dir"
-        fi
-
-        gatk --java-options "-Xmx${command_mem}m" Funcotator \
-            --data-sources-path $DATA_SOURCES_FOLDER \
-            --ref-version ${reference_version} \
-            -R ${ref_fasta} \
-            -V ${m2_vcf} \
-            -O ${output_vcf} \
-            ${"--transcript-selection-mode " + transcript_selection_mode} \
-            ${transcript_selection_arg}${default="" sep=" --transcript-list " transcript_selection_list} \
-            ${annotation_def_arg}${default="" sep=" --annotation-default " annotation_defaults} \
-            ${annotation_over_arg}${default="" sep=" --annotation-override " annotation_overrides} \
-            ${filter_funcotations_args}
-    >>>
-
-    runtime {
-        docker: gatk_docker
-        bootDiskSizeGb: 12
-        memory: machine_mem + " MB"
-        disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
-        preemptible: select_first([preemptible_attempts, 3])
-        cpu: select_first([cpu, 1])
-    }
-
-    output {
-        File funcotated_vcf = "${output_vcf}"
-        File funcotated_vcf_index = "${output_vcf_index}"
-    }
-}
-
+task FuncotateMaf {
+     # inputs
+     File ref_fasta
+     File ref_fasta_index
+     File ref_dict
+     File input_vcf
+     File input_vcf_idx
+     String reference_version
+     String output_format = "MAF"
+     String? sequencing_center
+     String? sequence_source
+     String case_id
+     String? control_id
+
+     File? data_sources_tar_gz
+     String? transcript_selection_mode
+     File? transcript_selection_list
+     Array[String]? annotation_defaults
+     Array[String]? annotation_overrides
+     Boolean filter_funcotations
+     File? interval_list
+
+     String? extra_args
+
+     # ==============
+     # Process input args:
+     String annotation_def_arg = if defined(annotation_defaults) then " --annotation-default " else ""
+     String annotation_over_arg = if defined(annotation_overrides) then " --annotation-override " else ""
+     String filter_funcotations_args = if (filter_funcotations) then " --remove-filtered-variants " else ""
+     String interval_list_arg = if defined(interval_list) then " -L " else ""
+     String extra_args_arg = select_first([extra_args, ""])
+     String final_output_filename = basename(input_vcf, ".vcf") + ".maf.annotated"
+     # ==============
+
+     # runtime
+
+     String gatk_docker
+     File? gatk_override
+     Int? mem
+     Int? preemptible_attempts
+     Int? disk_space_gb
+     Int? cpu
+
+     Boolean use_ssd = false
+
+     # This should be updated when a new version of the data sources is released
+     # TODO: Make this dynamically chosen in the command.
```

Comment:
This was out of date and has been eliminated in favor of a reasonable default.  Also, we will probably remove the FTP download at some point, since it often breaks.

Done